testing/scenario_app/.gitignore

```diff
@@ -0,0 +1,5 @@
+.packages
+pubspec.lock   # This only has local dependencies in it.
```

Comment:
This might need some elaboration. I think every reader will end up asking.

Reply:
Which part?  Why we're excluding pubspec.lock, or what it means that it only has local dependencies?